### PR TITLE
fix: error

### PR DESCRIPTION
**Pull request by schmelto**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -64,19 +63,18 @@ let ans = undefined;
...
 //call every time page is refreshed
 
-function loadQuiz(){
-    let currentQuizData = quizData[currentQuestion] //keeps track of current quiz page being displayed
-    question.innerHTML = currentQuizData.question;
-    a_text.innerText = currentQuizData.a;
-    b_text.innerText = currentQuizData.b;
-    c_text.innerText = currentQuizData.c;
-    d_text.innerText = currentQuizData.d;
-    currentQuestion++;
```
Comment (schmelto) [inline index.js:74]: This line caused the problem

**Review by sauradip007 — APPROVED** [review]
Awesome ! 😄 Really appreciate you helping out there

File: index.js [hunk 2]
```diff
@@ -96,28 +94,21 @@ function getSelected(){
...
-       
+submitbtn.addEventListener("click", () => {
+  let answer = getSelected();
+  console.log(answer);
+  if (answer) {
+    let currentQuizData = quizData[currentQuestion];
+    if (answer === currentQuizData.correct) {
+      score++;
+    }
+    currentQuestion++;
```
Comment (schmelto) [inline index.js:105]: Here you increased the counter another time

File: index.js (hunk 1 above)
Comment (sauradip007) [reply index.js:74]: ahh so it kept double increasing within the event listener as well.Thanks ! 👍

File: index.js (hunk 2 above)
Comment (sauradip007) [reply index.js:105]: got it

**Pull request merged**